builtin/providers/template/resource.go

```diff
@@ -87,6 +94,8 @@ func Read(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+type TemplateRenderError error
```

Comment:
Don't think this needs to be exported, since we only return it from a private function?